content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,40 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. In the
+documentation the workspace is the _user visible_ part of the directory that
+contains all your <abbr>project</abbr> files e.g. raw datasets, source code, ML
+models, etc. Users work in this directory using their data and model files that
+and manipulate the contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
+`dvc import-url`). Changes to the files, directories, notebooks, models and in
+general machine learning file system can be tracked (`dvc commit`) and versioned
+in Git (`dvc checkout`). <abbr>Pipelines</abbr> and <abbr>dependencies</abbr>
+between them can be defined. Data and model files can be moved to the cloud
+(`dvc remote`) and retrieved one by one when necessary (`dvc push`, `dvc pull`).
+They can be removed (`dvc remove`) from the workspace. DVC supports all typical
+operations of files and directories of a file system through its commands.
+
+Behind the scene these operations of a <abbr>DVC project</abbr> uses
+<abbr>metafiles</abbr> to track the content and dependencies.
```

Comment:
Good idea to mention these but maybe mention .dvc and dvc.yaml files specifically instead, as examples of what the workspace can contain (in the main paragraph).

```suggestion
```

Reply:
Mentioning files by name makes me nervous 😅 

I added the names to the last paragraph in 46fe4bba but if you think it's better to remove the paragraph completely, I can just do that as well. Thanks.

Reply:
Will make a seprate comment. Resolving this one.